R/parser.R

```diff
@@ -539,10 +539,11 @@ parse_github_replies <- function(github_replies_folder_path){
 #'
 #' @param depends_jar_path path to depends jar
 #' @param git_repo_path path to git repo (ends in .git)
+#' @param output_dir path to output directory (formatted output_path/)
 #' @param language the language of the .git repo (accepts cpp, java, ruby, python, pom)
 #' @export
 #' @family parsers
-parse_dependencies <- function(depends_jar_path,git_repo_path,language){
+parse_dependencies <- function(depends_jar_path,git_repo_path,language,output_dir="/tmp/"){
```

Comment:
~~I am not sure why this was added.~~

I believe I asked you to parameterize tmp. Could you refer me to the comment I asked this so I remember why?

Reply:
Hi Carlos, here's the relevant comment:
https://github.com/sailuh/kaiaulu/issues/168#issuecomment-1478886665

Reply:
:) Thank you! Yes, this is perfect.